addons/project/models/res_partner.py

```diff
@@ -28,3 +31,22 @@ def _compute_task_count(self):
                 if partner in self:
                     partner.task_count += group['partner_id_count']
                 partner = partner.parent_id
+
+    def _create_portal_users(self):
+        partners_without_user = self.filtered(lambda partner: not partner.user_ids)
+        if not partners_without_user:
+            return self.env['res.users']
+        template_user_id = literal_eval(self.env['ir.config_parameter'].sudo().get_param('base.template_portal_user_id', 'False'))
```

Comment:
Surely `int` would be sufficient?

Reply:
Yeah, it would be sufficient except in the case where we did not find the `base.template_portal_user_id` but we could change the default value to a digit like 0 for instance.
By the way, this code has been removed to directly use the [`_create_user_from_template` method in `res.users`](https://github.com/odoo/odoo/blob/c795927ce1e1c26b30ff1332def2318377828ccc/addons/auth_signup/models/res_users.py#L134-L152).